stripe/stripe_object.py

```diff
@@ -225,12 +223,31 @@ def refresh_from(
         self._transient_values = self._transient_values - set(values)
 
         for k, v in values.items():
-            super(StripeObject, self).__setitem__(
-                k,
-                util.convert_to_stripe_object(
-                    v, api_key, stripe_version, stripe_account
-                ),
-            )
+            inner_class = self._get_inner_class_type(k)
+            is_dict = self._get_inner_class_is_beneath_dict(k)
+            if is_dict:
+                obj = {
+                    k: util.convert_to_stripe_object(
+                        v,
+                        api_key,
+                        stripe_version,
+                        stripe_account,
+                        None,
+                        inner_class,
+                    )
+                    for k, v in v.items()
+                    if v is not None
```

Comment:
are we completely filtering out Nones?

Reply:
Yes, probably doesn't make a difference but I will fix.

Reply:
Fixed in [Don't filter out Nones](https://github.com/stripe/stripe-python/pull/1061/commits/8b1d5cd56c2818b601acc8d142e59c5af4153a9f).